maintainers/scripts/copy-tarballs.pl

```diff
@@ -180,62 +180,65 @@ sub uploadFile {
             chomp $hash;
         }
 
-        if (defined $ENV{DEBUG}) {
-            print "$url $algo $hash\n";
-            next;
-        }
-
-        if ($url !~ /^http:/ && $url !~ /^https:/ && $url !~ /^ftp:/ && $url !~ /^mirror:/) {
-            print STDERR "skipping $url (unsupported scheme)\n";
-            next;
-        }
+        for my $url (@$urls) {
+            if (defined $ENV{DEBUG}) {
+                print "$url $algo $hash\n";
+                next;
+            }
 
-        if ($isPatch) {
-            print STDERR "skipping $url (support for patches is missing)\n";
-            next;
-        }
+            if ($url !~ /^http:/ && $url !~ /^https:/ && $url !~ /^ftp:/ && $url !~ /^mirror:/) {
+                print STDERR "skipping $url (unsupported scheme)\n";
+                next;
+            }
 
-        next if defined $exclude && $url =~ /$exclude/;
+            if ($isPatch) {
+                print STDERR "skipping $url (support for patches is missing)\n";
+                next;
```

Comment:
Likewise, `$isPatch` should be checked outside of the inner loop. Either that, or I should call `last`.